CHANGELOG.md

```diff
@@ -1301,7 +1301,7 @@ Another important change is the integration with Amazon Braket, such that Mitiq
 
 In the context of probabilistic error cancellation, the sub-module `mitiq.pec.representations` has been significantly improved. Now one can easily compute the optimal quasi-probabiliy representation of an ideal gate as a linear combination of `NoisyOperation` objects.
 
-Thanks to all contributors (@L-P-B, @Aaron-Robertson, @francespoblete, @LaurentAjdnik, @maxtremblay, @andre-a-alves, @paniash, @purva-thakre) and in particular to the participants of [unitaryHACK](https://unitaryfund.github.io/unitaryhack/)!
+Thanks to all contributors (@L-P-B, @Aaron-Robertson, @francespoblete, @LaurentAjdnik, @maxtremblay, @andre-a-alves, @paniash, @purva-thakre) and in particular to the participants of [unitaryHACK](https://unitaryhack.dev/)!
```

Comment:
```suggestion
Thanks to all contributors (@L-P-B, @Aaron-Robertson, @francespoblete, @LaurentAjdnik, @maxtremblay, @andre-a-alves, @paniash, @purva-thakre) and in particular to the participants of [unitaryHACK](https://2021.unitaryhack.dev/)!
```

Reply:
@jordandsullivan Did you want to add this change?

Reply:
Ah yes, I mistook your suggested change for a commit. Just reverted, added the above change, and created a new PR here: #2246.